zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -1738,29 +1705,36 @@ public Stat writeDataGetStat(final String path, Object datat, final int expected
     return writeDataReturnStat(path, datat, expectedVersion);
   }
 
-  public void asyncCreate(final String path, Object datat, final CreateMode mode,
```

Comment:
Why changing the name? I think datat stands for data type. In this scenario, it means this is an object with a certain data type. "data" is very generic so it might be less meaningful.
I don't have a strong preference on the names. But if you are going to change it, please change all the datat in this java file at least : ) Or even better, the whole project.

Reply:
In that case, please change them in a batch in a separate PR. This change won't help with this PR.

Reply:
Since there is not much value to change them, I would just leave them as they are: `datat`